.github/workflows/linting.yaml

```diff
@@ -146,6 +158,18 @@ jobs:
         working-directory: src/worker
         run: uv run ruff check --output-format=github .
 
+      - name: Install pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 9
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: 'pnpm'
+          cache-dependency-path: |
+            src/gui/pnpm-lock.yaml
```

Comment:
I am not quite sure why is this needed in worker tests.
- `uv run pytest` does not seem to rely on `pnpm` in a way visible to me.